### PR TITLE
[fix bug 1228341] - Fix protocol relative links on annual report pages

### PR DESCRIPTION
**Pull request by alexgibson**
This fixes most links on these old pages (at least the insecure warnings introduced by the protocol relative URLs). Also explicitly makes links secure than can/should be.

### CODE REVIEW

File: bedrock/foundation/templates/foundation/annualreport/2009/broadening-our-scope.html [hunk 1]
```diff
@@ -137,7 +137,7 @@ <h4 class="section-title" id="apps">{{ _('"Apps."') }}</h4>
       <h4 class="section-title" id="services">{{ _('Sharing, Data and Services') }}</h4>
 
       <p>
-        {% trans f1="//f1.mozillamessaging.com/" %}
+        {% trans f1="http://f1.mozillamessaging.com/" %}
```
Comment (schalkneethling) [inline bedrock/foundation/templates/foundation/annualreport/2009/broadening-our-scope.html]: I guess this is one of those that just does not work anymore?

File: bedrock/foundation/templates/foundation/annualreport/2009/broadening-our-scope.html [hunk 2]
```diff
@@ -183,10 +183,10 @@ <h4 class="section-title" id="drumbeat">
...
           logo=static('img/foundation/annualreport/2009/drumbeat-logo.jpg'),
-          webcraft="//www.drumbeat.org/webcraft",
-          festival="//www.drumbeat.org/festival",
-          movies="//www.drumbeat.org/webmademovies",
-          subtitles="//www.drumbeat.org/universal-subtitles/",
+          webcraft="http://www.drumbeat.org/",
+          festival="http://www.drumbeat.org/",
+          movies="http://www.drumbeat.org/",
+          subtitles="http://www.drumbeat.org/",
           fund="https://sendto.mozilla.org/"
```
Comment (schalkneethling) [inline bedrock/foundation/templates/foundation/annualreport/2009/broadening-our-scope.html]: Should we change this to donate.mozilla.org while we are at it?

File: bedrock/foundation/templates/foundation/annualreport/2009/broadening-our-scope.html (hunk 1 above)
Comment (alexgibson) [inline bedrock/foundation/templates/foundation/annualreport/2009/broadening-our-scope.html]: Yes certain URLs just don't exist anymore. I'm not going to try and find fixes for all in this PR, just correct the links that are working but currently pointing to the wrong protocol.

File: bedrock/foundation/templates/foundation/annualreport/2009/broadening-our-scope.html (hunk 2 above)
Comment (alexgibson) [inline bedrock/foundation/templates/foundation/annualreport/2009/broadening-our-scope.html]: Can do :+1:

**Comment by alexgibson** [comment]
@schalkneethling - updated thanks

**Comment by alexgibson** [comment]
Hold please, spotted a few more in the 2011 report that need fixing too :/

**Comment by alexgibson** [comment]
OK updated again.

**Comment by schalkneethling** [comment]
r+ :link:

**Pull request merged**